test/test_index_management.py

```diff
@@ -94,6 +99,27 @@ def wait_for_ready(self, coll, name=_NAME, predicate=None):
                 break
             time.sleep(5)
 
+    def test_comment_field(self):
+        # Create a collection with the "create" command using a randomly generated name (referred to as ``coll0``).
+        coll0 = self.db[f"col{uuid.uuid4()}"]
+        coll0.insert_one({})
+
+        # Create a new search index on ``coll0`` that implicitly passes its type.
+        search_definition = {"mappings": {"dynamic": False}}
+        implicit_search_resp = coll0.create_search_index(
+            model={"name": _NAME + "-implicit", "definition": search_definition}
```

Comment:
can you test adding a `comment` to this call too?

Reply:
Done